app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -48,23 +50,24 @@ class ProfileFragment : Fragment() {
         startActivity(Intent(activity, MainActivity::class.java).setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP))
     }
 
-    override fun onAttach(context: Context) {
-        super.onAttach(context)
-        if (!profileViewModel.isLoggedIn()) {
-            Toast.makeText(context, "You need to Login!", Toast.LENGTH_LONG).show()
-            redirectToLogin()
-        }
-    }
-
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
         rootView = inflater.inflate(R.layout.fragment_profile, container, false)
+        CoordinatorLayout = rootView.findViewById(R.id.profileCoordinatorLayout)
 
         setHasOptionsMenu(true)
 
+        if (!profileViewModel.isLoggedIn()) {
+            Snackbar.make(
+                CoordinatorLayout, "You need to log in first!", Snackbar.LENGTH_SHORT).show()
+            Handler().postDelayed({
+                redirectToLogin()
+            }, 1000)
```

Comment:
Strange indentation

Reply:
will correct don't know why autoformat is going for this